manager/controllers/systems.go

```diff
@@ -86,6 +92,10 @@ type SystemsResponse struct {
 // @Param    filter[stale]           query   string  false "Filter"
 // @Param    filter[packages_installed] query string false "Filter"
 // @Param    filter[packages_updatable] query string false "Filter"
+// @Param    filter[stale_timestamp] query string false "Filter"
+// @Param    filter[stale_warning_timestamp] query string false "Filter"
+// @Param    filter[culled_timestamp] query string false "Filter"
+// @Param    filter[created] query string false "Filter"
```

Comment:
We defined new filters but no one is used in any test. Could you please add one test to systems_test.go (TestSystemsCreated) where the "created" filter will be used? It should be similar like TestSystemsWorkloads3. Just update one or more "created" values in test_data.sql - inventory.hosts items to have different value to test filter. Now all values are the same.